### ISSUE TITLE
Testing Joomla 4 - pgk_agosms not found

### ISSUE THREAD

**Issue by MaghSamana** [astridx/pkg_agosms#77]
In testing site, after migration to Joomla 4, there is a notification for updating this plugin. However the file is missing:

`Failed to download package. Download it and install manually from https://github.com/astridx/pkg_agosms/releases/download/v4.0.0/pkg-agosms-4.0.0.zip`

**Comment by astridx** [comment]
Thank you for your information. You are right.  I prepaired an update. But it is not ready. What extensions are you using exactly?

**Comment by MaghSamana** [comment]
3.0.1

I disabled it before the migration, but when I logged in Joomla 4 afterwards, the system had already found the update.

**Comment by MaghSamana** [comment]
Also, I have an old module in my files (from Joomla 1.x), which the developer had abandoned years ago. It consists only of an xml and php file with some parameters, and shows the sunrise/noon/sunset times of the current day for a given place.

I was able to do some changes and installed it in Joomla 3. Up to J 3.10 it was fine.
Now, though I get no errors about it in Joomla 4, its output no longer shows. Probably the module format needs to be a bit different for J4.

Would you like to take over?
There aren't such extensions around and it's very useful.
[mod_sunrise_sunset.zip](https://github.com/astridx/pkg_agosms/files/6666098/mod_sunrise_sunset.zip)

**Comment by astridx** [comment]
> @MaghSamana 
> 3.0.1, the module for Open street maps.
> 
> I disabled it before the migration, but when I logged in Joomla 4 afterwards, the system had already found the update.

I uploaded the version today. The update under Joomla 4 should now work via the Joomla Updater. 

I would be happy if you could test it and give me a short feedback.

If it does not work via the Joomla updater, you can find the Versin here: https://github.com/astridx/pkg_agosms/releases/tag/v4.0.0




> Also, I have an old module in my files (from Joomla 1.x), which the developer had abandoned years ago. It consists only of an xml and php file with some parameters, and shows the sunrise/noon/sunset times of the current day for a given place.
> Would you like to take over?
> There aren't such extensions around and it's very useful.

I'll have a look when I've finished my extensions.

**Comment by MaghSamana** [comment]
The installation didn't get through.

First, I went through the Updates panel. That was the error message: 
![Screenshot #004 - Extensions_ Update - Administration](https://user-images.githubusercontent.com/46110970/122655950-4387be80-d15f-11eb-9754-9f1ca2322a3a.png)

Then I tried the Install extension. Again, similar error:
![Screenshot #005 - Extensions_ Install - Administration](https://user-images.githubusercontent.com/46110970/122655962-61552380-d15f-11eb-953a-53c08d3a1036.png)

Test environment: 
Joomla 4.0.0-rc2
PHP 7.4.20

**Comment by astridx** [comment]
Thank you very much for testing.

I am sorry. But I realized, that I can't manage to provide an update version. 

There is an independent version for Joomla 4 that is very similar to this one under releases. I have now deleted the old non-functioning zip-files. 

If you are interested in the independent Joomla 4 version, you are welcome to take a look at the draft here:
https://github.com/astridx/pkg_agosms/releases/tag/v4.0.0
